# MoveIt-specific simulation settings
moveit_sim_hw_interface:
  joint_model_group: controllers_initial_group_
  joint_model_group_pose: controllers_initial_pose_
# Settings for ros_control control loop
generic_hw_control_loop:
  loop_hz: 300
  cycle_time_error_threshold: 0.01
# Settings for ros_control hardware interface
hardware_interface:
  joints:
    - hombro
    - codo
    - pris
  sim_control_mode: 1  # 0: position, 1: velocity
# Publish all joint states
# Creates the /joint_states topic necessary in ROS

controller_list:
  []
nelen:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  codo_control:
    type: effort_controllers/JointPositionController
    joint: codo
    pid: {p: 100.0, i: 0.01, d: 10.0}
  hombro_control:
    type: effort_controllers/JointPositionController
    joint: hombro
    pid: {p: 100.0, i: 0.01, d: 10.0}
  prism_control:
    type: effort_controllers/JointPositionController
    joint: pris
    pid: {p: 100.0, i: 0.01, d: 10.0}
